tests/test_glue/test_parser.sphinx3.xml

```diff
@@ -139,7 +139,7 @@
                 <inline classes="pasted-text">
                     undisplayed
                  inline…
-            <figure align="default" ids="abc" names="abc">
+            <figure align="default" ids="abc" names="abc" alt="alt-text for glued figure">
```

Comment:
From looking at the output html, it seems to be that the `alt` should actually be on the `image` tag two lines below, not the figure. The trick is coming up with a consistent way to pass that information through the various containers/transforms.